themes/mlibrary_new/custom.php

```diff
@@ -126,24 +100,36 @@ function mlibrary_new_display_popular_tags()
 
 
 function mlibrary_new_display_exhibit_card_info($exhibitPage)
-{  // to see if there is no image attached to the first page.
-  
+{ 
+   // to see if there is no image attached to the first page.
+  $page_description = '';
+  $page_title = get_view()->shortcodes(metadata($exhibitPage, 'title'));
+  $default_image = img("defaulthbg.jpg");
+  $page_image = "<img class='image-card' alt='' src='{$default_image}'/>";
```

Comment:
I recommend to refactor this into `mlibrary_new_get_page_image($rawAttachment)`.
Lines 107, 108, and 121-129 would go there.

Reply:
Working on this, since you can have a text block that has no image. So we have to have a separate function that tests all of these scenarios.